types/fundamental-react/fundamental-react-tests.tsx

```diff
@@ -2147,9 +2147,10 @@ const sideNavs = (
 
 const stepInputs = (
     <div>
-        <StepInput disabled value={10} />
-        <StepInput readOnly value={10} />
+        <StepInput disabled stepValue={10} />
```

Comment:
where you see a `stepValue` ?
https://github.com/SAP/fundamental-react/blob/master/src/StepInput/StepInput.js